pkg/kosmosctl/manifest/manifest_secrets.go

```diff
@@ -0,0 +1,21 @@
+package manifest
+
+const (
+	ClusterTreeClusterManagerSecret = `---
```

Comment:
What is the function of this secret?
Would it be better to switch to a time-limited bearer token? @OrangeBao

Reply:
The HTTPS service will use this certificate, not for clients.